src/primitives/felt.test.ts

```diff
@@ -1,20 +1,15 @@
 import { test, expect, describe } from 'bun:test';
 import { Felt } from './felt';
-import { OutOfRangeBigInt, PrimitiveError } from 'errors/primitives';
-import { Relocatable } from './relocatable';
+import { OutOfRangeBigInt } from 'errors/primitives';
 
 describe('Felt', () => {
   describe('constructor', () => {
     test('should throw if initializing a felt with a negative inner', () => {
```

Comment:
do we really want to throw in this case? isn't just wrapping to `PRIME - n`?

Reply:
It is better to wrap around PRIME at constructor level.
Plus it allows simplifying the different operator methods (add, sub, mul)